test/games/strategy/engine/data/UnitCollectionTest.java

```diff
@@ -0,0 +1,386 @@
+  final int unitCountDefaultPlayerUnitTypeTwo = getUnitsOfDefaultPlayerOfUnitTypeTwo().size();
+  final int unitCountDefaultPlayer = unitCountDefaultPlayerUnitTypeOne + unitCountDefaultPlayerUnitTypeTwo;
+  final Unit unitDefaultPlayer1 = new Unit(unitTypeOne, defaultPlayerID, gameData);
+  final Unit unitDefaultPlayer2 = new Unit(unitTypeTwo, defaultPlayerID, gameData);
+  final Unit unitDefaultPlayer3 = new Unit(unitTypeTwo, defaultPlayerID, gameData);
+
+  private Collection<Unit> getUnitsOfDefaultPlayerOfUnitTypeOne() {
+    final Collection<Unit> units = new ArrayList<Unit>();
+    units.add(unitDefaultPlayer1);
+    return units;
+  }
+
+  private Collection<Unit> getUnitsOfDefaultPlayerOfUnitTypeTwo() {
+    final Collection<Unit> units = new ArrayList<Unit>();
+    units.add(unitDefaultPlayer2);
+    units.add(unitDefaultPlayer3);
+    return units;
+  }
+
+  final int unitCountOtherPlayerUnitTypeOne = 3;
+  final int unitCountOtherPlayerUnitTypeTwo = 4;
+  final int unitCountOtherPlayer = unitCountOtherPlayerUnitTypeOne + unitCountOtherPlayerUnitTypeTwo;
+  final Unit unitOtherPlayer1 = new Unit(unitTypeOne, otherPlayerID, gameData);
+  final Unit unitOtherPlayer2 = new Unit(unitTypeOne, otherPlayerID, gameData);
+  final Unit unitOtherPlayer3 = new Unit(unitTypeOne, otherPlayerID, gameData);
+  final Unit unitOtherPlayer4 = new Unit(unitTypeTwo, otherPlayerID, gameData);
+  final Unit unitOtherPlayer5 = new Unit(unitTypeTwo, otherPlayerID, gameData);
+  final Unit unitOtherPlayer6 = new Unit(unitTypeTwo, otherPlayerID, gameData);
+  final Unit unitOtherPlayer7 = new Unit(unitTypeTwo, otherPlayerID, gameData);
+
+
+  final int unitCountUnitTypeOne = unitCountDefaultPlayerUnitTypeOne + unitCountOtherPlayerUnitTypeOne;
+  final int unitCountUnitTypeTwo = unitCountDefaultPlayerUnitTypeTwo + unitCountOtherPlayerUnitTypeTwo;
+
+  private Collection<Unit> getUnitsOfOtherPlayerOfUnitTypeOne() {
+    final Collection<Unit> units = new ArrayList<Unit>();
+    units.add(unitOtherPlayer1);
+    units.add(unitOtherPlayer2);
+    units.add(unitOtherPlayer3);
+    return units;
+  }
+
+  private Collection<Unit> getUnitsOfOtherPlayerOfUnitTypeTwo() {
+    final Collection<Unit> units = new ArrayList<Unit>();
+    units.add(unitOtherPlayer4);
+    units.add(unitOtherPlayer5);
+    units.add(unitOtherPlayer6);
+    units.add(unitOtherPlayer7);
+    return units;
+  }
+
+  private void addAllPlayerUnits() {
+    addAllDefaultPlayerUnits();
+    addAllOtherPlayerUnits();
+  }
+
+  private void addAllDefaultPlayerUnits() {
+    uc.addAllUnits(getUnitsOfDefaultPlayerOfUnitTypeOne());
+    uc.addAllUnits(getUnitsOfDefaultPlayerOfUnitTypeTwo());
+  }
+
+  private void addAllOtherPlayerUnits() {
+    uc.addAllUnits(getUnitsOfOtherPlayerOfUnitTypeOne());
+    uc.addAllUnits(getUnitsOfOtherPlayerOfUnitTypeTwo());
+  }
+
+  @Test
+  public void unitCollection() {
+    assertThat(uc.getHolder(), is(equalTo(defaultPlayerID)));

```

Comment:
'is' and then 'equalTo' are redundant. you can do simply `assertThat(.., is(defaultPlayerID))`